executor/batch_checker.go

```diff
@@ -292,6 +292,50 @@ func (b *batchChecker) deleteDupKeys(ctx context.Context, sctx sessionctx.Contex
 	return nil
 }
 
+// getOldRow gets the table record row from storage for batch check.
```

Comment:
getOldRow  -> getOldRowNew

Reply:
Lint tool would not complain about it, because we don't force the check.
This function will be renamed to `getOldRow`, and I'm clean up the old `getOldRow` function.